.github/workflows/core.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Setup Bundler
       run: scripts/setup_bundler.sh
     - name: Build and test
-      run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }}
+      run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseCore.podspec --skip-tests --platforms=${{ matrix.target }}
```

Comment:
@paulb777 since I know Storage skipped the tests, I was trying to compare the `core.yml` with the `storage.yml`. Is this a valid change too? Im not too sure how all of these workflows play together yet

Reply:
This is not a valid change. Storage runs its unit tests from `pod gen'd` workspace since some of its tests require secrets. If the same is now true for Core, we would also need to set up tests from `pod gen`.